### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by aliyevorkhan** [mevo12318/emotion_detection#1]
Hi Juan, I want to know, how I can get best accuracy? Increase data or something? I want to improve and use this.

**Comment by juan-csv** [comment]
Hi Orkhan, it occurs to me that you can do the following to improve the accuracy of the model:
1. You are right, adding the dataset can always improve the accuracy of the model, you could use these 3 datasets
https://www.kaggle.com/jonathanoheix/face-expression-recognition-dataset
https://www.kaggle.com/ahmedmoorsy/facial-expression
https://www.kaggle.com/shawon10/ckplus

2. You can remove or merge the classes that have few labels, for example the dataset you train with has few disgust labels, you could merge these labels with angry.

3. Try a more complex architecture, you could add one or two more layers to the network or try with pre-trained models and use transfer learning.

I leave you the architecture of the network that I use in case you want to work on it.

# instantiate model
model = Sequential()

# 1 - Convolution
model.add(Conv2D(64,(3,3), padding='same', input_shape=(48, 48,1)))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 2nd Convolution layer
model.add(Conv2D(128,(5,5), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 3rd Convolution layer
model.add(Conv2D(512,(3,3), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 4th Convolution layer
model.add(Conv2D(512,(3,3), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# Flattening
model.add(Flatten())

# Fully connected layer 1st layer
model.add(Dense(256))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(Dropout(0.25))

# Fully connected layer 2nd layer
model.add(Dense(512))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(Dropout(0.25))

model.add(Dense(num_classes, activation='softmax'))

model.summary()

**Comment by aliyevorkhan** [comment]
Dear Juan, thanks for your reply. I will try all of this to improve accuracy. For now I close this issue. After tries I will open this again and I will share what happen.
